### DISCUSSION TITLE
Application says a new version is available while there's none

### DISCUSSION THREAD

**Discussion by Porkepix** [wireapp/wire-desktop#5080] [Ideas]
Wire version: Version 3.16.2923
Wire for web version: Wire for Web Version 2020.04.07.1425
Operating system: ArchLinux. Rolling release.

What steps will reproduce the problem?

1.  Just have Wire launched

What is the expected result?
Have the "new update" banner only when there's indeed a new update.

What is the actual result?
The banner is shown even when using the last version.

According to GitHub's releases, this version is the most recent one for Linux.
However, version numbers differ depending on the OS, so maybe it does see there are higher version number for mac or Windows. It's still latest Linux' version though.

The banner shown:
![Screenshot from 2020-04-21 18-53-06](https://user-images.githubusercontent.com/709500/79891538-660a3480-8401-11ea-9ff3-2410e9932d1c.png)

**Comment by ffflorian** [comment]
This banner tells you that there is a new version of Wire for Web available. It will be loaded when you click "update" or after restarting the app.

**Comment by Porkepix** [comment]
@ffflorian Can't the message be clarified then? If I get it correctly from your explanation, Wire the application is mainly a shell/container and run another app in the app? It's not really obvious for the user as you that this banner is meant for an internal component and rather refer to application's version itself.

**Comment by ffflorian** [comment]
@Porkepix thanks for the suggestion. I forwarded it to our team.

**Comment by lucendio** [comment]
IMHO, this issue should stay open until it's been fixed, not forwarded.

**Comment by vincegre** [comment]
Good luck if you expect any improvement from Wire (using that app since ages and it's still as bad as first day nearly :-1:
